torch/nn/modules/pooling.py

```diff
@@ -510,6 +501,15 @@ class AvgPool1d(_AvgPoolNd):
         [torch.FloatTensor of size (1,1,3)]
     """
 
+    def __init__(self, kernel_size, stride=None, padding=0, ceil_mode=False,
+                 count_include_pad=True):
+        super(AvgPool1d, self).__init__()
+        self.kernel_size = _single(kernel_size)
+        self.stride = _single(stride if stride is not None else kernel_size)
```

Comment:
The breakage happens when exporting AvgPool to onnx. Tests in https://github.com/onnxbot/onnx-fb-universe caught this, however the CI was turned off since last Friday's repo merge so the failure is not reported to the original PR. I'm moving all these tests to this repo in a separate PR #6223